<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sonic Gesture</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/thumb_up.png</normaloff>:/icons/icons/thumb_up.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_Video"/>
    <addaction name="actionOpen_Device_2"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionFinder_View"/>
    <addaction name="actionCapture_View"/>
    <addaction name="actionSource_view"/>
   </widget>
   <widget class="QMenu" name="menuRecord">
    <property name="title">
     <string>&amp;Record</string>
    </property>
    <addaction name="actionRecord_visual_output"/>
    <addaction name="actionRecord_original_movie"/>
    <addaction name="separator"/>
    <addaction name="actionStart_Record"/>
    <addaction name="actionStop_record"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="actionAbout_Sonic_Gesture"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_View"/>
   <addaction name="menuRecord"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Video"/>
   <addaction name="actionOpen_Device_2"/>
   <addaction name="separator"/>
   <addaction name="actionFinder_View"/>
   <addaction name="actionCapture_View"/>
   <addaction name="actionSource_view"/>
   <addaction name="separator"/>
   <addaction name="actionRecord_original_movie"/>
   <addaction name="actionRecord_visual_output"/>
   <addaction name="separator"/>
   <addaction name="actionStart_Record"/>
   <addaction name="actionStop_record"/>
  </widget>
  <action name="actionOpen_Video">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/film</normaloff>:/icons/film</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Video...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionOpen_Device_2">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/camera</normaloff>:/icons/camera</iconset>
   </property>
   <property name="text">
    <string>Open &amp;Device...</string>
   </property>
  </action>
  <action name="actionFinder_View">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/finder</normaloff>:/icons/finder</iconset>
   </property>
   <property name="text">
    <string>Finder View</string>
   </property>
  </action>
  <action name="actionCapture_View">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/capture</normaloff>:/icons/capture</iconset>
   </property>
   <property name="text">
    <string>Capture View</string>
   </property>
  </action>
  <action name="actionRecord_original_movie">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/record input</normaloff>:/icons/record input</iconset>
   </property>
   <property name="text">
    <string>Record original input</string>
   </property>
  </action>
  <action name="actionRecord_visual_output">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/record output</normaloff>:/icons/record output</iconset>
   </property>
   <property name="text">
    <string>Record sonic output</string>
   </property>
  </action>
  <action name="actionSource_view">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/normal</normaloff>:/icons/normal</iconset>
   </property>
   <property name="text">
    <string>Source view</string>
   </property>
  </action>
  <action name="actionAbout_Sonic_Gesture">
   <property name="text">
    <string>About Sonic Gesture</string>
   </property>
  </action>
  <action name="actionStart_Record">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/start record</normaloff>:/icons/start record</iconset>
   </property>
   <property name="text">
    <string>Start record</string>
   </property>
  </action>
  <action name="actionStop_record">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/stop record</normaloff>:/icons/stop record</iconset>
   </property>
   <property name="text">
    <string>Stop record</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
